code/obj/machinery/singularity.dm

```diff
@@ -554,7 +551,7 @@ for some reason I brought it back and tried to clean it up a bit and I regret ev
 		T2 = T
 		var/obj/machinery/containment_field/CF = new/obj/machinery/containment_field/(src, G) //(ref to this gen, ref to connected gen)
 		CF.set_loc(T)
-		CF.set_dir(field_dir)
+		CF.dir = field_dir
```

Comment:
There was an effort to use set_dir over other operations, this resolve something that helper does not?

Reply:
Look through 278c2a7, there are... a number of things I hope you didn't mean to change.

Reply:
Literally every single one of them, *scream

Reply:
👍